fe/fe-core/src/test/java/com/starrocks/sql/plan/DecodeRewriteTest.java

```diff
@@ -386,6 +386,7 @@ public void testDecodeNodeRewrite12() throws Exception {
 
     @Test
     public void testDecodeNodeRewrite13() throws Exception {
+        FeConstants.runningUnitTest = true;
```

Comment:
why need change this config?

Reply:
let olap scan node partition is not zero(which is default in UT), this will impact the join node type